docs/ops/deployment/yarn_setup.md

```diff
@@ -251,6 +250,29 @@ The user-jars position in the class path can be controlled by setting the parame
 - `FIRST`: Adds the jar to the beginning of the system class path.
 - `LAST`: Adds the jar to the end of the system class path.
 
+## Run an application in Application Mode
+
+To launch an application in [Application Mode]({{ site.baseurl }}/ops/deployment/#deployment-modes), you can type:
```

Comment:
links should use `{% link %}` syntax, so `{% link ops/deployment/index.md %}#deployment-modes`